ide/libs.graalsdk/external/binaries-list

```diff
@@ -14,5 +14,14 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-8E62643FD621053E00EE52D797C7FBD08D7FDEDC org.graalvm.sdk:graal-sdk:20.3.0
-8968BDBE4058F4E6610AF0DC184BD885B236A2B1 org.graalvm.sdk:launcher-common:20.3.0
+8AA4ECE060629AAAED57215A5FD8C377EFF1A191 org.graalvm.sdk:graal-sdk:24.0.0
```

Comment:
Technically, `graal-sdk` may not be needed - it seems to be just a placeholder package, and the original packages probably have been split up into some of the other bundles here.

As a side note, maybe there's a better name for the module? maybe `libs.graal`?

Reply:
I would prefer not to rename modules, unless necessary. We can have module_auto_deps entry, I'd stick with the current name.